binary/spdx/testdata/json-format.spdx.json

```diff
@@ -1,7 +1,7 @@
 {
   "spdxVersion": "SPDX-2.3",
   "dataLicense": "CC0-1.0",
-  "SPDXID": "SPDXRef-Document",
+  "SPDXID": "SPDXRef-DOCUMENT",
```

Comment:
This is correct :+1: 
Ref: [SPDX 2.3 - Document identifier](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#63-spdx-identifier-field)